lib/IndicatorManager.vala

```diff
@@ -313,14 +316,21 @@ public class Wingpanel.IndicatorManager : GLib.Object {
     public void register_indicator (string path, Wingpanel.Indicator indicator) {
         debug ("%s registered", indicator.code_name);
 
+        var deregister_map = new Gee.HashMap<string,Wingpanel.Indicator> ();
```

Comment:
Just a nitpick, but you should add a space after the `,` :smile: 

Other than that, great job. It does fix the warnings i was getting